index.njk

```diff
@@ -5,32 +5,27 @@ title: Rehydrate
 
 <section class="middle">
   <div class="podcast-summary">
-    <p>A podcast where we read books together at a TV series-like pace.</p>
-    <p>Books include <i>The Three-Body Problem</i> and <i>The Long Way to a Small, Angry Planet</i></p>
+    <p>A podcast where we discuss books, music, and other media</p>
   </div>
 
   <section id="how-to-listen">
     <div class="listening-directions">
-      Point your podcast app at the <a href="rss.xml">RSS feed</a> to listen!
```

Comment:
A lot of podcast listeners listen via RSS, but I don't think they know what to do with just the text "RSS". We need some verbal call to action here.